recipes/cocotb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 1f390bb6468d70e04acebeb9ff05b54be42cbbbbbb3bd6c75d1ec0a36cb97e23
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - '*/libgcc_s.so.1'
+    - '*/libstdc++.so.6'
```

Comment:
This package contains compiled C code. Why would you whitelist missing C libraries?